model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -937,6 +937,8 @@ async def execute(
 
         request_id = str(uuid4())
         add_trace_request_id(request_id)
+        if request.top_k == 0:  # top_k can't be 0, only takes >= 1, or -1/None to disable top_k
+            request.top_k = -1
```

Comment:
can you add a check for validity of vllm/tgi/lightllm specific parameters? Otherwise the args will pass through silently and users will not know what happened.

Reply:
+1, also we should centralize all of the framework-specific validation, like later in this function.